src/modules/CoreUtils.js

```diff
@@ -8,12 +8,19 @@ class CoreUtils {
     this.w = ctx.w
   }
 
-  static checkComboSeries(series) {
+  static checkComboSeries(series, chartType) {
     let comboCharts = false
     let comboBarCount = 0
     let comboCount = 0
-
-    // if user specified a type in series too, turn on comboCharts flag
+    
+    if (chartType === 'undefined') {
```

Comment:
undefined as a string should be checked as `typeof chartType === 'undefined'`

Reply:
Thanks. Done